server/scratchy_server/test/test_message.tavern.yaml

```diff
@@ -0,0 +1,180 @@
+---
+test_name: test the message most basic function
+
+# one stage for each request
+stages:
+  - name: Post a room
+    request:
+      url: http://localhost:5000/api/room
+      json:
+        title: my title
+        description: my description
+      method: POST
+      headers:
+        content-type: application/json
+    response:
+      status_code: 200
```

Comment:
It's already the default value [according to the documentation](status_code), you can remove it to make the yaml cleaner and only specify it when needed.